File: Data Structures & Algorithms/products-of-array-discluding-self/submission-1.py
from collections import defaultdict
class Solution:
    def productExceptSelf(self, nums: List[int]) -> List[int]:
        num_map = defaultdict(int)
        result = [] 

        for i in nums:
            num_map[tuple(nums)].append[i]

        for value in range(i+1, len(nums)):
            product = num_map.get(value)
            result.append(product)

File: Data Structures & Algorithms/concatenation-of-array/submission-1.java
class Solution {
    public int[] getConcatenation(int[] nums) {
        int[] nums_out = new int[len(nums) * 2];
        
        for(int i = 0; i < len(nums) * 2; i++) {
            int j = 0;
            if (j == len(nums) - 1) {
                j = 0;
            }

            nums_out[i] = nums[j];
        }
    }
}